omod/src/main/webapp/fragments/patientdashboard/editVisitDatesDialog.gsp

```diff
@@ -21,7 +21,7 @@
                         defaultDate: config.defaultStartDate
                 ])}
             </p>
-            <% if(config.defaultEndDate != null) { %>
```

Comment:
Why change this from `defaultEndDate` to `visitEndDate`?

Reply:
This was really a hack to allow the user to have a null defaultEndDate  yet at the same time preventing the user setting an end date on an Active visit which was already in place....